### PR TITLE
Normalizeurl cleanup

### PR DESCRIPTION
**Pull request by rfrenchseti**
Normalizeurl hits the real world. "Missing field" messages are annoying and not useful. Page temporarily replaces startobs until the new branch is ready. Handle case of empty URL without an error message. Clean up some other messages. Fix some bugs. Add cart_startobs.

This checkin needs Dave's normalizeurl branch to be merged in first.

Oh, note that there are a LOT of test failures with this change due to the page/startobs swap. If you put that one part back, all tests pass.

### CODE REVIEW

File: opus/application/static_media/js/opus.js [hunk 1]
```diff
@@ -41,18 +41,20 @@ var opus = {
...
+        "cols": default_columns.split(","),  // default result table columns by slug
+        "widgets": [], // search tab widget columns
+        "order": default_sort_order.split(","),  // result table ordering
+        "view": "search", // search, browse, cart, detail
+        "browse": "gallery", // either 'gallery' or 'data'
+        "cart_browse": "gallery",  // which view is showing on the cart page, gallery or data
+        "startobs": 1, // for this branch it will not get updated
+        "cart_startobs": 1, // for this branch it will not get updated
+        "detail": "", // opus_id of detail page content
+        "page": default_pages,  // what page are we on, per view, default defined in header.html
```
Comment (dstopp) [inline opus/application/static_media/js/opus.js:55]: can you add a comment that this will go away w/the startobs branch integration?

**Review by dstopp — APPROVED** [review]

File: opus/application/apps/ui/views.py [hunk 2]
```diff
@@ -660,7 +666,7 @@ def _escape_or_label_results(old_slug, pi):
                 msg = ('Search query for "'
                        + _escape_or_label_results(search2, pi2)
                        + '" maximum had an illegal '
-                       'value; it has been ignored.')
+                       +'value; it has been ignored.')
```
Comment (juzen2003) [inline opus/application/apps/ui/views.py:669]: - Just to sync up code style with previous line, maybe need a space after "+" ?

**Review by juzen2003 — APPROVED** [review]

File: opus/application/apps/ui/views.py (hunk 2 above)
Comment (rfrenchseti) [reply opus/application/apps/ui/views.py:669]: I made the spacing consistent.

File: opus/application/static_media/js/opus.js (hunk 1 above)
Comment (rfrenchseti) [reply opus/application/static_media/js/opus.js:55]: I don't understand this comment. What will go away? startobs? page? cart_startobs?

**Pull request merged**